src/grad.jl

```diff
@@ -1,40 +1,64 @@
 export grad, jacobian, jvp, j′vp, to_vec
+function replace_arg(x, xs::Tuple, k::Int)
```

Comment:
style: blank line before function

 ```suggestion

function replace_arg(x, xs::Tuple, k::Int)
```